### ISSUE TITLE
nearley-test needs a 'quiet' or silent flag

### ISSUE THREAD

**Issue by sandiegoscott** [Hardmath123/nearley#127]
...so I can see just the result without the parsing details.

**Comment by sandiegoscott** [comment]
But piping through tail is not bad, though the number of lines of output needs to be known:

```
$ nearley-test -i "#00ff00" csscolor.js | tail -n 2
Parse results: 
[ [ '#', [ '0' ], [ '0' ], [ 'f' ], [ 'f' ], [ '0' ], [ '0' ] ] ]
```

**Comment by Hardmath123** [comment]
If all you want is to see the results, you should be able to write yourself a simple program that does that for you, right? nearley-test's main purpose is to show the parsing details; the output is just added information.

**Comment by bobbybee** [comment]
@Hardmath123 All I've ever used nearley-test for has been parsing output (to -test- the grammar). This suggestion might still not be necessary, but a --quiet flag might be nice?

**Comment by Hardmath123** [comment]
\*shrug\*

Okay. Can you PR this? Should be an easy fix.

**Comment by tjvr** [comment]
Looks like this was fixed! :)  #130
